bot/main.py

```diff
@@ -46,6 +57,11 @@ async def on_ready():
         activity=activity,
     )
 
+    while True:
+        await asyncio.sleep(spam_log_file_clear_interval)
+        with open(spam_log_file, "r+") as spam_log:
+            spam_log.truncate(0)
+
```

Comment:
Can this be done in the cog?

Ideally `main` does not care about the implementation of the log, and the responsibility for managing the log is entirely in the spam cog/module.